### ISSUE TITLE
Tide tutorial sample results different

### ISSUE THREAD

**Issue by temanashort** [crux-toolkit/crux-toolkit#522]
When following the Tide tutorial from the Crux page (http://crux.ms/tutorials/search.html), after running the two commands given in the tutorial: `crux tide-index small-yeast.fasta yeast-index` and `crux tide-search --compute-sp T demo.ms2 yeast-index`, I noticed that only 26 target-decoy pairs are matched with spectra.

Even after noticing that the default precursor window type and and window length have changed to ppm and 50 from mass and 3 and overriding the new defaults with `crux tide-search --compute-sp T --overwrite T --precursor-window-type mass --precursor-window 3 demo.ms2 yeast-index`, I found that the largest target Xcorr score from my output was 0.9251438 (much lower than 5.20757 which was the largest Xcorr from the tutorial results).

Additionally, the PSM that scored 5.20757 in the tutorial only scored 0.26897098 in my tide-search output. And other columns that were different (for this specific PSM) were delta_cn (0.936932 in tutorial, 0.22916 in my output), sp score, and b/y ions matched, which were actually 0.0 or 0 for every single data row in my output.

I wonder if there are other defaults for tide-index and/or tide-search that have changed since the tutorial was released which are making my results significantly different from the tutorial results, and/or if I should try the Tide tutorial on another computer. Also I noted that the `sort-by-column` function doesn't seem to exist anymore. My Crux version is 4.0-b854e9b6-2021-06-01, and I'm working on macOS.

**Comment by bmx8177** [comment]
Hi, thanks for reaching out.

There is one more parameter whose default value has changed. We changed ```--mz-bin-width 1.0005079``` to ```--mz-bin-width 0.02```. Changing that parameter back to the original value should make the output match what is shown on the tutorial. 

Yes, you are correct. We recently removed the ```sort-by-column``` and several other functions that were not commonly used. Thanks for pointing that out. We will update the tutorial to reflect that. 

Andy
